modules/claims_api/config/schemas/v2/526.json

```diff
@@ -13,9 +13,9 @@
   "properties": {
     "claimDate": {
       "description": "Date of claim submission to be applied to signature on form 21-526EZ. Date cannot be in the future.",
-      "pattern": "^(?:[0-9]{4})-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2][0-9]|3[0-1])(?:T[0-1][0-9]:[0-5][0-9]:[0-5][0-9](?:[-+][0-9]{2}:[0-9]{2}|Z))*$",
       "type": ["string", "null"],
-      "example": "2018-06-04",
+      "example": "2018-08-28T19:53:45+00:00",
```

Comment:
Just curious, since `request.json` is the one that displays the docs, do we need this timestamp in that to reflect the update?  Or is there a reason for it to be in `526.json` and `submission.json` but _not_ `request.json`?